packages/ses/src/compartment.js

```diff
@@ -269,6 +270,12 @@ export const makeCompartmentConstructor = (
 
     assign(globalObject, endowments);
 
+    // freeze Symbol, Compartment constructor and evaluators in new Compartments
+    freeze(globalObject.Symbol);
```

Comment:
You are correct, freezing this has no meaning, because `%SharedSymbol%` is being frozen already on https://github.com/endojs/endo/blob/master/packages/ses/src/lockdown.js#L386.
Meaning, this attempt should be removed completely, not moved over, since again, it's already being taken care of.

Reply:
83c41f8724b9d7b51e16237d22b580ad83612327